.gitignore

```diff
@@ -58,20 +54,17 @@ test_*_tempdir
 
 # Tests
 /python/test/unit/**/*.xml
```

Comment:
`$ find . -name '*.xml'` gives me
```
./python/test/unit/io/xml_value_collection_ref.xml
./python/test/unit/fem/gmsh_unit_interval.xml
./python/test/unit/fem/tetrahedron.xml
```

I `grep`-ed for the first two and I can't find them every being used.